contracts/interfaces/IFlowsService.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.16;
+
+interface IFlowsService {
+    /// @notice Claims rewards for the caller by transferring them from the LiquidityMining contract to the PowerToken contract.
+    /// @param lpTokens An array of LP tokens for which the rewards are to be claimed.
+    /// @dev This function calls the `claim` function of the `ILiquidityMiningV2` contract to retrieve the rewards amount to transfer.
+    /// It then adds the staked tokens to the `powerToken` contract and transfers the rewards from the `liquidityMining` contract to the `powerToken` contract.
+    /// @dev Reverts if the `lpTokens` array is empty.
+    /// @dev Reverts if there are no rewards to claim.
+    function claimPowerToken(address[] calldata lpTokens) external;
+
+    /// @notice Updates the indicators for a given account and LP tokens.
+    /// @param account The account address for which the indicators are to be updated.
+    /// @param lpTokens An array of LP tokens for which the indicators are to be updated.
+    /// @dev This function calls the `updateIndicators` function of the `ILiquidityMiningV2` contract to update the indicators.
+    /// @dev Reverts if the `lpTokens` array is empty.
+    function updateIndicators(address account, address[] calldata lpTokens) external;
```

Comment:
Indicators is quite generic name which can correspond to amm module so please add to this method some power token context for example: updateLiquidityMiningIndicators

Reply:
I changed it before your comment ;)